includes/azure-app-configuration-role-assignment.md

```diff
@@ -18,3 +18,4 @@ Assign the proper App Configuration role assignments to the credentials being us
 1. Select the service principal associated with the service connection that you created in the previous section.
 
     :::image type="content" source="./media/azure-app-configuration-role-assignment/add-role-assignment.png" alt-text="Screenshot shows the Add role assignment dialog.":::
+1. If the store contains Key Vault references, go to relevant Key Vault and assign **Key Vault Secret User** role to the service principal created in the previous step. From the Key Vault menu, select **Access policies** and ensure [Azure role-based access control](https://docs.microsoft.com/en-us/azure/key-vault/general/rbac-guide?tabs=azure-cli#enable-azure-rbac-permissions-on-key-vault) is selected as the permission model. 
```

Comment:
Looks good. Please update the link to be a relative link.

Reply:
Also please remove "en-us", that sets the language to english. If you remove it, it will auto set the language based on their preferences

Reply:
Removed "en-us" from the link and updated the link to "https://docs.microsoft.com/azure/key-vault/general/rbac-guide?tabs=azure-cli" just to confirm is this (https://docs.microsoft.com/azure/key-vault/general/rbac-guide?tabs=azure-cli")the relevant link?